manual/README.md

```diff
@@ -0,0 +1,27 @@
+# The pytket User Manual
+
+The manual is built from several reStructuredText (.rst) files in the manual directory.
+
+The manual can be built locally as follows.
+
+First clone the pytket repository
+
+```shell
+git clone git@github.com:CQCL/pytket.git
+```
+
+It is recommended to set up a virtual environment in the pytket repository to manage dependencies.
+
+Once the virtual environment is set up we can run the `build-manual` script from the pytket directory which installs the necessary python packages and builds the html pages.
+
+```shell
+./build-manual
```

Comment:
Are there any limitations to the operating systems on which this is working?

Reply:
There may be issues with pytket-qsharp on mac possibly? QSharp does not appear in the manual (only the examples) so it probably shouldn't be installed as a dependency here.